src/main/features/claimant-response/views/part-payment-received.njk

```diff
@@ -0,0 +1,45 @@
+{% extends "layout.njk" %}
+{% from "form.njk" import csrfProtection, radioGroup, saveAndContinueButton, errorSummary %}
+{% from "table.njk" import tableStart, tableEnd, row, rowWithMultipleValue %}
+
+{% set heading =  "Has the defendant paid you " + paidAmount | numeral +  "?" %}
+
+{% block content %}
+  <div class="grid-row">
+    <div class="column-two-thirds">
+      <form method="post" class="analytics-submit-event-trigger"
+            data-event-action="Claimant Response: Has defendant paid less than claim amount"
+            data-event-label-from="received"
+            novalidate>
+        {{ csrfProtection(csrf) }}
+
+        {{
+        radioGroup(
+          label = "Has the defendant paid you £" + paidAmount + "?",
```

Comment:
will `paidAmount` be properly formatted? Could 123.4 be rendered instead of 123.40?

Reply:
compare settle-claim
https://github.com/hmcts/cmc-citizen-frontend/pull/849/files?utf8=%E2%9C%93&diff=split&w=1#diff-f7025a8f3d68168f5cf1b8a942d00aadR22